Sources/SmithyStreams/FileStream.swift

```diff
@@ -26,7 +26,7 @@ public final class FileStream: Stream {
     let fileHandle: FileHandle
 
     /// Returns the current position of the stream.
-    public var position: Data.Index
+    public private(set) var position: Data.Index
```

Comment:
_Technically_ this is a breaking API change but it would cause a bug to alter this property from outside this type.